src/registrar/templates/portfolio_organization_info.html

```diff
@@ -42,49 +43,50 @@ <h1>Organization</h1>
             <p>The name of your organization will be publicly listed as the domain registrant.</p>
 
             {% if has_edit_portfolio_permission %}
-                <p>
-                    Your organization name can’t be updated here.
-                    To suggest an update, email <a href="mailto:help@get.gov" class="usa-link">help@get.gov</a>.
-                </p>
-    
-                {% include "includes/required_fields.html" %}
-                <form class="usa-form usa-form--large desktop:margin-top-4" method="post" novalidate>
-                    {% csrf_token %}
-                    <h4 class="margin-bottom-05">Organization name</h4>
-                    <p class="margin-top-0">
-                        {{ portfolio.federal_agency }}
-                    </p>
-                    {% input_with_errors form.address_line1 %}
-                    {% input_with_errors form.address_line2 %}
-                    {% input_with_errors form.city %}
-                    {% input_with_errors form.state_territory %}
-                    {% with add_class="usa-input--small" sublabel_text="Enter a 5-digit or 9-digit zip code, like 12345 or 12345-6789." %}
-                        {% input_with_errors form.zipcode %}
-                    {% endwith %}
-                    <button type="submit" class="usa-button">
-                    Save
-                    </button>
-                </form>
-            {% else %}
-                <h4 class="margin-bottom-05">Organization name</h4>
+            <p>
+                Your organization name can’t be updated here.
+                To suggest an update, email <a href="mailto:help@get.gov" class="usa-link">help@get.gov</a>.
+            </p>
+            {% endif %}
+            <h2>Organization admins</h2>
+            <ul class="usa-list usa-list--unstyled margin-top-2 margin-bottom-0">
+            {% for admin in portfolio_admins %}
+                <li>
+                    <p class="margin-bottom-0">{{admin.first_name}} {{admin.last_name}}</p>
+                    <p class="margin-top-0">{{admin.email}}</p>
+                </li>    
+            {% endfor %}
+            </ul>
+            
+            <h2>Organization name and address</h2>
+            
+            {% if has_edit_portfolio_permission %}
+            {% include "includes/required_fields.html" with remove_margin_top=True %}
+            <form class="usa-form usa-form--large" method="post" novalidate>
+                {% csrf_token %}
+                <p class="margin-bottom-05 text-primary-darker text-bold">Organization name</p>
                 <p class="margin-top-0">
-                    {{ portfolio.federal_agency }}
```

Comment:
Turns out we should be getting `organization_name` here, not `federal_agency`. Since `organization_name` appears to be optional in the model, I account for the case where it doesn't exist.